markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -100,6 +100,8 @@ contract PerpsAccountModule is IPerpsAccountModule {
         FeatureFlag.ensureAccessToFeature(Flags.PERPS_SYSTEM);
         Account.exists(accountId);
 
+        AsyncOrder.checkPendingOrder(accountId);
```

Comment:
Calling `checkPendingOrder()` will _throw_ if an order exists that has not yet expired. However, if the order has expired, the debt can still be _safely paid_ despite the existing order.

This behavior is perfectly fine, but integrators referring to the interface might not expect an error to be thrown when calling this function simply because an order exists.

I recommend that functions that throw errors via a nested call to another function should include that information in the interface.

For example: `f(x) -> g(x)` where `g(x)` may revert with `e`; `e` should be mentioned within the interface containing `f(x)`

> I encountered this issue myself while working from the integrator’s perspective 😅